src/Layers/BasemapLayer.js

```diff
@@ -183,6 +183,25 @@ export var BasemapLayer = TileLayer.extend({
           maxZoom: 19,
           attribution: 'Esri, DigitalGlobe, GeoEye, Earthstar Geographics, CNES/Airbus DS, USDA, USGS, AeroGRID, IGN, and the GIS User Community'
         }
+      },
+      WorldPhysical: {
+        urlTemplate: tileProtocol + '//{s}.arcgisonline.com/arcgis/rest/services/World_Physical_Map/MapServer/tile/{z}/{y}/{x}',
+        options: {
+          minZoom: 1,
+          maxZoom: 8,
+          subdomains: ['server', 'services'],
+          attribution: 'U.S. National Park Service'
+        }
+      },
+      WorldPhysicalLabels: {
```

Comment:
i'd skip adding `PhysicalLabels`. folks can just use `TerrainLabels` (which points at the same service).